Formula/openslide.rb

```diff
@@ -28,9 +36,18 @@ class Openslide < Formula
     sha256 "ed92d5a9f2e86df67640d6f92ce3e231419ce127131697fbbce42ad5e002c8a7"
   end
 
+
   def install
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}"
+    if build.head?
+      system "autoreconf", "-i"
+      system "./configure", "--without-makeinfo",
```

Comment:
Instead of these branches, let's do something like:
```ruby
system "autoreconf", "-i" if build.head?

args = %w[
--disable-dependency-tracking
--disable-static
--prefix=#{prefix}
]
args << "--without-makeinfo" if build.head?
```
(Why is that last change required for a HEAD build, though?